Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -406,14 +447,13 @@ def construct_slack_msg(triggering_workflow: str,
         # No color is needed in case of success, as it's controlled by the color of the test failures' indicator.
 
     title += title_append
-    title += f' - @{CI_SERVER_HOST}' if CI_SERVER_HOST else ''
```

Comment:
Why did you remove this?

Reply:
the XDR migration is over, no longer needed.